draft-nygren-dnsop-svcb-httpssvc.md

```diff
@@ -557,16 +559,16 @@ to avoid additional latency in comparison to ordinary AAAA/A lookups.
 
 ## Clients using a Proxy
 
-Clients using a proxy via issuing CONNECT requests
-({{!RFC7231}} Section 4.3.6) should follow the same
-resolution for resolving SVCB records.  If a ServiceForm SVCB
-record is found, the client should select the highest priority acceptable
-option.  From this the client should construct the request-target
-used in the CONNECT request.
-
-In cases where no ServiceForm SVCB record is found, the client should
-proceed as normal and use HOST to construct the request-target.
+Clients using a domain-oriented transport proxy like HTTP CONNECT
+({{!RFC7231}} Section 4.3.6) or SOCKS5 ({{!RFC1928}}) SHOULD disable
+SVCB support if performing SVCB queries would violate the
+client's privacy intent.
```

Comment:
Mention "unless SVCB queries are also issued via the proxy (eg, via DoH)"?

Part of the need to specify the proxy behavior is to get ESNI to work properly.  Should we spell this out here (to explain why this proxy behavior is important), or perhaps later?  ie, part of getting ESNI to work means the keys and SvcDomainName corresponding to the connect need to match up.

Reply:
OK, I added an explanation of motivations for the proxy section.

Reply:
Sounds good.  Do we want to highlight that proxies trying to do content filtering and assuming that what they are connecting to is the Host/SNI could get confused?  (Which are the same proxies that ESNI will make sad.)

Reply:
I'd rather leave that alone if possible.  Using an "affiliated resolver" as described would cover that use case.